### ISSUE TITLE
No audible feedback for dropdown menus

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1628]
There is a button labeled ‘Files’ located in the top left corner that toggles the navigation side menu. However, there is no audible feedback provided by screen readers to indicate whether the menu has been opened or closed. This lack of feedback can create confusion for screen
reader users, as they are unable to determine whether or not their action had any effect. To resolve this issue, the button should indicate the current state of the menu (e.g.,‘menu opened’ or ‘menu closed’).
